test/BaselineOutput/Common/EntryPoints/core_manifest.json

```diff
@@ -7675,7 +7675,7 @@
           "Required": false,
           "SortOrder": 5.0,
           "IsNullable": false,
-          "Default": null
```

Comment:
**Question:** Just curious, did you have to manually change this? Or was it automatically updated simply by adding the constructor logic you added?